python/hail/utils/misc.py

```diff
@@ -41,6 +42,21 @@ def new_temp_file(n_char=10, prefix=None, suffix=None):
                             'MEMORY_AND_DISK', 'MEMORY_AND_DISK_2', 'MEMORY_AND_DISK_SER',
                             'MEMORY_AND_DISK_SER_2', 'OFF_HEAP')
 
+_test_dir = None
+
+
```

Comment:
some extra lines above and below

Reply:
this is from autoformat, so I think this obeys PEP 8